README.md

```diff
@@ -121,7 +121,9 @@ The current implementation of the JSON API uses DataStax Enterprise (DSE) as the
 CI tests are run using the Stargate and JSON API versions specified in the [api-compatibility.versions](api-compatibility.versions) file.
 
 ## Connecting to AstraDB
-Integration with [AstraDB](https://astra.datastax.com/) is experimental and is not usable/ fully ready at this time.
+Here's a quick start guide on how to connect to AstraDB using `stargate-mongoose` driver.
```

Comment:
I would strongly prefer we have at least a small example here that highlights the `isAstra` option and `createAstraUri()`. The quickstart guide is good, but I think it is also beneficial to demonstrate minimum viable connection code in the README. I think the following example would be sufficient:

```ts
const mongoose = require("mongoose");
const { driver, createAstraUri } = require("stargate-mongoose");

const uri = createAstraUri(
  process.env.ASTRA_DB_ID,
  process.env.ASTRA_DB_REGION,
  process.env.ASTRA_DB_KEYSPACE,
  process.env.ASTRA_DB_APPLICATION_TOKEN,
);

mongoose.setDriver(driver);

await mongoose.connect(uri, {
  isAstra: true,
});
```

Reply:
Agreed. Added the above code snippet along with the AstraDB docs link in [this commit](https://github.com/stargate/stargate-mongoose/pull/164/commits/6196cec70378b041b8245fa9813d170ad010dea0).